modules/ROOT/pages/index.adoc

```diff
@@ -39,6 +39,7 @@ Both the public and private content in Exchange can consist of the following kin
 * Connectors - Packaged connectivity to an endpoint developed and deployed on Anypoint Platform with third-party APIs and standard integration protocols. Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.
 * Templates - Packaged integration patterns built on best practices to address common use cases. You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
 * Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution.
+* Policies - Configuration modules for managing APIs.
```

Comment:
Policies - Configuration modules that extend the functionality of an API and enforces certain capabilities such as security.

Reply:
Much better. Fixed. Thank you!